File: Single-Linked-List/create_node.py
# Definition of Node
class Node:
    def __init__(self, data):
        self.data = data
        self.next = None

# Function to create a new node
def create_node():
    value = int(input("Enter data for the node: "))
    new_node = Node(value)
    return new_node

File: Single-Linked-List/CreateNode.java
import java.util.Scanner;

// Definition of Node
class Node {
    int data;
    Node next;

    // Constructor to create a node
    Node(int data) {
        this.data = data;
        this.next = null;
    }
}

public class CreateNode {

    // Method to create a new node
    public static Node createNode() {
        Scanner sc = new Scanner(System.in);

        System.out.print("Enter data for the node: ");
        int value = sc.nextInt();

        Node newNode = new Node(value);
        return newNode;
    }
}
